assets/app/views/_overview-deployment.html

```diff
@@ -78,7 +78,7 @@
     </div>
 
     <!-- Pods donut chart and scaling controls -->
-    <div column>
+    <div column class="overview-pods">
```

Comment:
Thanks @benjaminapetersen, I created a follow on issue. I think the change should be made separate from this PR.